app/src/main/java/ch/epfl/sdp/musiconnect/CurrentUser.java

```diff
@@ -8,18 +8,10 @@
 public class CurrentUser {
     // static variable single_instance of type Singleton
     private static CurrentUser single_instance = null;
+    public final String email;
```

Comment:
Maybe the email field can be private ?

Reply:
I need to access it. I'll change it to private and add a getter. thx

Reply:
but after re-checking, it's a final field, so it won't be changed from outside. so no risk there. I think I'm gonan leave it this way :p